fri/server/main.py

```diff
@@ -70,14 +70,17 @@ def build(dir):
     makestudy_dir = dirname + "/" + graphml_file   #for makestudy
     dir_path = os.path.abspath(os.path.join(concore_path, graphml_file)) #path for ./build
     if not os.path.exists(dir_path):
-        proc = call(["./makestudy", makestudy_dir], cwd=concore_path)
+        proc = call(["makestudy", makestudy_dir],shell=True, cwd=concore_path)
         if(proc == 0):
             resp = jsonify({'message': 'Directory successfully created'})
             resp.status_code = 201
         else:
             resp = jsonify({'message': 'There is an Error'})
             resp.status_code = 500        
-    call(["./build"], cwd=dir_path)   
+    else:

```

Comment:
Please keep the PRs to be minimal. If you are addressing a bug, the PR should just address that, instead of making the logs more verbose, like here.

Reply:
Actually I put an else block here as if
If block is not executed then resp object will not be defined and will give error , as resp object is only defined inside IF block

Reply:
aha, sorry, my bad. You are right.